### ISSUE TITLE
[Module Proposal]: `avm-ptn-finopstoolkit-finopshub`

### ISSUE THREAD

**Issue by didayal-msft** [Azure/Azure-Verified-Modules#983]
### Check for previous/existing GitHub issues/module proposals

- [X] I have checked for previous/existing GitHub issues/module proposals.

### Check this module doesn't already exist in the module indexes

- [X] I have checked for that this module doesn't already exist in the module indexes.

### Bicep or Terraform?

Terraform

### Module Classification?

Pattern Module

### Module Name

avm-ptn-finopstoolkit-finopshub

### Module Details

Terraform module for FinOps toolkit's hub.
Details here: https://microsoft.github.io/finops-toolkit/hubs/template

### Do you want to be the owner of this module?

Yes

### Module Owner's GitHub Username (handle)

didayal-msft

**Comment by prjelesi** [comment]
This will be on hold until we get confirmation from FinOps toolkit team. @flanakin for visibility.

**Comment by didayal-msft** [comment]
Yes @prjelesi this needs to be on hold as per last week's discussion. We will wait for bicep pattern module and see demand as well as review overhead of managing 2 modules. For now this can be put on hold or closed as per procedure.

**Comment by flanakin** [comment]
I believe @sebassem is already working on this 🙂

**Comment by sebassem** [comment]
We already published this module :)

https://github.com/Azure/bicep-registry-modules/tree/main/avm/ptn/finops-toolkit/finops-hub

**Comment by prjelesi** [comment]
@sebassem as i can see you published bicep module, this is request for terraform. @flanakin @didayal-msft

**Comment by prjelesi** [comment]
Since we agreed to proceed with this:

Hi @didayal-msft 

Thanks for requesting/proposing to be an AVM module owner!

We just want to confirm **you agree to the below pages** that define what module ownership means:

- [Team Definitions & RACI](https://azure.github.io/Azure-Verified-Modules/specs/shared/team-definitions)
- [Shared Specification (Bicep & Terraform)](https://azure.github.io/Azure-Verified-Modules/specs/shared)
- [Module Support](https://azure.github.io/Azure-Verified-Modules/help-support/module-support)

Any questions or clarifications needed, let us know!

If you agree, please just **reply to this issue with the exact sentence below** (as this helps with our automation 👍):

"I CONFIRM I WISH TO OWN THIS AVM MODULE AND UNDERSTAND THE REQUIREMENTS AND DEFINITION OF A MODULE OWNER"

Thanks,

The AVM Core Team

#RR

**Comment by didayal-msft** [comment]
I CONFIRM I WISH TO OWN THIS AVM MODULE AND UNDERSTAND THE REQUIREMENTS AND DEFINITION OF A MODULE OWNER

**Comment by prjelesi** [comment]
Hi @didayal-msft 

Thanks for confirming that you wish to own this AVM module and understand the related requirements and responsibilities!

Before starting development, please ensure ALL the following requirements are met.

**Please use the following values explicitly as provided in the [module index](https://azure.github.io/Azure-Verified-Modules/indexes/) page**:

- For your module:
  - `ModuleName` - for naming your module
  - `TelemetryIdPrefix` - for your module's [telemetry](https://azure.github.io/Azure-Verified-Modules/specs/shared/#id-sfr3---category-telemetry---deploymentusage-telemetry)
- For your module's repository:
  - Repo name and folder path are defined in `RepoURL`
  - Create GitHub teams for module owners and contributors and grant them permissions as outlined [here](https://azure.github.io/Azure-Verified-Modules/specs/shared/#id-snfr20---category-contributionsupport---github-teams-only).
  - Grant permissions for the AVM core team and PG teams on your GitHub repo as described [here](https://azure.github.io/Azure-Verified-Modules/specs/shared/#id-snfr9---category-contributionsupport---avm--pg-teams-github-repo-permissions).

Check if this module exists in the other IaC language. If so, collaborate with the other owner for consistency. 👍

You can now start the development of this module! ✅ Happy coding! 🎉

**Please respond to this comment and request a review from the AVM core team once your module is ready to be published! Please include a link pointing to your PR, once available. 🙏**

Any further questions or clarifications needed, let us know!

Thanks,

The AVM Core Team

**Comment by arthurclares** [comment]
This module will be deprecated. Please, use the ARM template for deployment: aka.ms/finops/hubs/deploy . All the future requests for support should be sent to https://github.com/microsoft/finops-toolkit/issues
